src/test/java/core/basesyntax/MyTest.java

```diff
@@ -0,0 +1,88 @@
+package core.basesyntax;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class MyTest {
+    private User user;
+    private String login;
+    private int loginLength;
+    private String password;
+    private int age;
+    private int passwordLength;
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        login = user.getLogin();
+        password = user.getPassword();
+        age = user.getAge();
+        loginLength = user.getLogin().length();
+        passwordLength = user.getPassword().length();
+    }
+
+    @Test
+    void checkNotNullUser() {
+        if (user == null) {
+            throw new MyUncheckedException("User is null");
+        }
+    }
+
+    @Test
+    void loginIsNotNull() {
+        assertNotNull(login);
+    }
+
+    @Test
+    void loginLengthIsNotZero() {
+        if (loginLength == 0) {
+            throw new MyUncheckedException("Login is invalid");
+        }
+    }
+
+    @Test
+    void loginLengthIsMoreThan_6() {
+        if (loginLength < 6) {
+            throw new MyUncheckedException("Login should be bigger");
+        }
+    }
+
+    @Test
+    void passwordLengthIsNotZero() {
+        if (passwordLength == 0) {
+            throw new MyUncheckedException("Password is invalid");
+        }
+    }
```

Comment:
The method `passwordLengthIsNotZero` should use `assertTrue(passwordLength > 0)` instead of manually throwing `MyUncheckedException`. The method name should also follow the naming convention.